web/src/pages/PluginTemplate/Create.tsx

```diff
@@ -108,7 +108,12 @@ const Page: React.FC = (props) => {
 
           {step === 1 && <Step1 form={form1} />}
           {step === 2 && (
-            <PluginPage initialData={plugins} onChange={setPlugins} schemaType="route" />
+            <PluginPage
```

Comment:
I have never done `.tsx` before, but when I try to do HTML attributes I like to order them alphabetically.

Suggested change:

```
<PluginPage
    initialData={plugins}
    onChange={setPlugins}
    referPage="route"
    schemaType="route"
    />
```

Reply:
yep

Reply:
thanks for your useful advice, I have updated the code, PTAL when you are free. @justinmclean